bin/pycbc_multi_inspiral

```diff
@@ -86,89 +110,254 @@ psd.insert_psd_option_group_multi_ifo(parser)
+    #been loud in at least 2 detectors.
+    coinc_idx = np.unique(coinc_list,return_counts=True)[0][
+                   np.unique(coinc_list,return_counts=True)[1]>1]
+    return coinc_idx
+
 strain_dict = strain.from_cli_multi_ifos(opt, opt.instruments, 
                                          dyn_range_fac=DYN_RANGE_FAC)
-strain_segments_dict = strain.StrainSegments.from_cli_multi_ifos(opt, 
-                                                  strain_dict, opt.instruments)
-
+strain_segments_dict = strain.StrainSegments.from_cli_multi_ifos(
+                         opt, strain_dict, opt.instruments)
 with ctx:
     fft.from_cli(opt)
-
     # Set some often used variables for easy access
     flow = opt.low_frequency_cutoff
     flow_dict = defaultdict(lambda : flow)
     for count, ifo in enumerate(opt.instruments):
         if count == 0:
-            sample_rate = strain_dict[ifo].sample_rate
-            sample_rate_dict = defaultdict(lambda : sample_rate)
-            flen = strain_segments_dict[ifo].freq_len
-            flen_dict = defaultdict(lambda : flen)
-            tlen = strain_segments_dict[ifo].time_len
-            tlen_dict = defaultdict(lambda : tlen)
-            delta_f = strain_segments_dict[ifo].delta_f
-            delta_f_dict = defaultdict(lambda : delta_f)            
-        else:
-            try:
-                assert(sample_rate == strain_dict[ifo].sample_rate)
-                assert(flen == strain_segments_dict[ifo].freq_len)
-                assert(tlen == strain_segments_dict[ifo].time_len)
-                assert(delta_f == strain_segments_dict[ifo].delta_f)
-            except:
-                err_msg = "Sample rate, frequency length and time length "
-                err_msg += "must all be consistent across ifos."
-                raise ValueError(err_msg)
+	    sample_rate = strain_dict[ifo].sample_rate
+	    sample_rate_dict = defaultdict(lambda : sample_rate)
+	    flen = strain_segments_dict[ifo].freq_len
+	    flen_dict = defaultdict(lambda : flen)
+	    tlen = strain_segments_dict[ifo].time_len
+	    tlen_dict = defaultdict(lambda : tlen)
+	    delta_f = strain_segments_dict[ifo].delta_f
+	    delta_f_dict = defaultdict(lambda : delta_f)            
+	else:
+	    try:
+		assert(sample_rate == strain_dict[ifo].sample_rate)
+		assert(flen == strain_segments_dict[ifo].freq_len)
+		assert(tlen == strain_segments_dict[ifo].time_len)
+		assert(delta_f == strain_segments_dict[ifo].delta_f)
+	    except:
+		err_msg = "Sample rate, frequency length and time length "
+		err_msg += "must all be consistent across ifos."
+		raise ValueError(err_msg)
 
     logging.info("Making frequency-domain data segments")
-    segments = {}
-    for ifo in opt.instruments:
-        segments[ifo] = strain_segments_dict[ifo].fourier_segments()
-        del strain_segments_dict[ifo]
-
+    segments = {ifo : strain_segments_dict[ifo].fourier_segments() for ifo in opt.instruments}
+    del strain_segments_dict
+    """
```

Comment:
Please remove, rather than comment, unnecessary code. (Use the `#` character when commenting code).